docs/account/managing-your-profile.md

```diff
@@ -15,7 +15,10 @@ To manage your profile information such as your name and avatar, click on your a
 
 ## Deleting your account
 
-When you delete your account on Codacy, your profile and information about your personal repositories will be completely removed from Codacy.
+When you delete your account on Codacy:
+
+-   Your profile and information about your personal repositories will be completely removed from Codacy
+-   Codacy will [stop analyzing any repositories added to Codacy using your account](../faq/troubleshooting/why-did-codacy-stop-commenting-on-pull-requests.md)
```

Comment:
This will give more visibility that, at least as of now, Codacy will stop analyzing repositories that were added/integrated using the deleted member's account.

Reply:
I like it! In practice commits still analyse, and just PRs will not show, but to be honest I would leave it as you stated to not make it more confusing. As in practice, if the user leaves, different things get broken, so I like saying it like this

Reply:
These extra details are good to know @machadoit, but I also agree it's best to omit them in the public documentation.

I added a comment on that line linking back to this pull request, just to give awareness in case we need to make further adjustments in the future.